### ISSUE TITLE
Enhance the UI or at leat use better Icons

### ISSUE THREAD

**Issue by jv-barsuk** [fransschreuder/libosmscout#29]
I like OSM Scout but I think the UI could be enhanced a bit and be adapted to fit the Ubuntu Phone style more. 

Appart from that I think that the icons in the menu a difficult to understand. My comments to them from left to right:
1. Ok but not great, what about something like this: http://tinyurl.com/pde573u
2. I have no clue what it does
3. make it monocrome, its hard to tell what it means. Use an icon like this: 
![north_arrow](https://cloud.githubusercontent.com/assets/2843476/8635085/6231d668-2814-11e5-8050-af5899f31891.png) / the second one could be the arrow now used at the first icon.
4. thats good and understandable
5. ok, too, just make it bold and a bit bigger
6. also good

If you want help in creating the icons I can help you.

**Comment by fransschreuder** [comment]
Right, I see you are volunteering to create artwork?

**Comment by jv-barsuk** [comment]
Yes I would. I'll try what I can do in the next weeks.

**Comment by Markcortbass** [comment]
Unfortunately, I'm not a programmer and can't help you with this.. But would it possible to use "Page With Bottom Edge"? It would be so awesome if you could implement this in your GPS navigation application
http://ubuntu-component-store.readthedocs.org/en/latest/_components/pagewithbottomedge.html

**Comment by fransschreuder** [comment]
Hello Markcortbass, I don't know what you are talking about, the bottom edge actions are there.

**Comment by Markcortbass** [comment]
Hi Frans Schreuder,

I was talking about the 'official' bottom edge. Like we have in the bottom of the clock application.

**Comment by fransschreuder** [comment]
Yes, In that case I need to rewrite the top qml as a page, which it isn't at the moment.
